### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by rajadilipkolli**

### CODE REVIEW

File: mediaservices/azure-mediaservices-spring-boot-starter/src/main/java/com/microsoft/azure/autoconfigure/mediaservices/MediaServicesProperties.java [hunk 1]
```diff
@@ -28,12 +28,12 @@
     /**
      * Media service account name.
      */
-    @NotEmpty
+    @NotEmpty(message = "Please Set accountName in application.properties")
```
Comment (ZhijunZhao) [inline mediaservices/azure-mediaservices-spring-boot-starter/src/main/java/com/microsoft/azure/autoconfigure/mediaservices/MediaServicesProperties.java]: accountName might confuse user. How about azure.mediaservices.accountName?

File: mediaservices/azure-mediaservices-spring-boot-starter/src/main/java/com/microsoft/azure/autoconfigure/mediaservices/MediaServicesProperties.java [hunk 2]
```diff
@@ -28,12 +28,12 @@
...
     private String accountName;
     /**
      * Media service account key.
      */
-    @NotEmpty
+    @NotEmpty(message = "Please Set accountKey in application.properties")
```
Comment (ZhijunZhao) [inline mediaservices/azure-mediaservices-spring-boot-starter/src/main/java/com/microsoft/azure/autoconfigure/mediaservices/MediaServicesProperties.java]: same here.

File: documentdb/spring-data-azure-documentdb/src/main/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbOperations.java [hunk 3]
```diff
@@ -15,7 +15,7 @@
 
     String getCollectionName(Class<?> entityClass);
 
-    <T> DocumentCollection createCollection(String collectionName, RequestOptions collectionOptions);
+    DocumentCollection createCollection(String collectionName, RequestOptions collectionOptions);
```
Comment (ZhijunZhao) [inline documentdb/spring-data-azure-documentdb/src/main/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbOperations.java]: Thanks for fixing all these typos.

File: mediaservices/azure-mediaservices-spring-boot-starter/src/main/java/com/microsoft/azure/autoconfigure/mediaservices/MediaServicesProperties.java (hunk 1 above)
Comment (rajadilipkolli) [reply mediaservices/azure-mediaservices-spring-boot-starter/src/main/java/com/microsoft/azure/autoconfigure/mediaservices/MediaServicesProperties.java]: implemented the same

**Pull request merged**